test/helpers/set-fixture-hooks.js

```diff
@@ -1,7 +1,4 @@
-import { startServer, setup, teardown } from './setup'
+import { setup, teardown } from './setup'
 
 export default chosenFixture =>
-  chosenFixture
-    .before(startServer)
-    .beforeEach(setup)
-    .afterEach(teardown)
+  chosenFixture.beforeEach(setup).afterEach(teardown)
```

Comment:
This is missing a wait stage though, either `setup` or something in the Jenkinsfile have to wait for the application to have started before tests can be run. The waiting condition would be that the port 3000 is open, or that `/ping` succeeds?

Reply:
So the setup no longer starts the server. It truncates the tables and starts the mock s3 server... mmm ... I wonder if we need the fakes3?